contracts/OpenExecutionManager.sol

```diff
@@ -41,7 +41,7 @@ contract OpenExecutionManager is IExecutionManager, AccessControlExtended {
     function isExecutor(
         bytes32 packedMessage,
         bytes memory sig
-    ) external view override returns (address executor, bool isValidExecutor) {
```

Comment:
Keep it view so that same interface applies to both execution managers

Reply:
But we do have an interface where it is view and they both are implementing it. Pure is more stricter so just affecting this contract.